src/bindings/string-conversion.cc

```diff
@@ -14,6 +14,12 @@ optional<u16string> string_conversion::string_from_js(Local<Value> value) {
   u16string result;
   result.resize(string->Length());
   string->Write(
+
+    // Nan doesn't wrap this functionality
+    #if NODE_MAJOR_VERSION >= 12
+          Isolate::GetCurrent(),
+    #endif
```

Comment:
In node 12, the `v8::String::Write` method changed to take a `v8::Isolate` as its first parameter. Usually we don't have to manually deal with different Node/V8 versions in our code, because `Nan` wraps these APIs for us. But `Nan` does not provide a wrapper for `String::Write`.